css/src/assets/styles/styles.css

```diff
@@ -0,0 +1,162 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #F5F5F5;
+  background-size: 414px 347px;
+}
+#mainDiv {
```

Comment:
I'd recommend avoiding applying of styles to `id`'s. Using `id`'s for styles can cause specificity headaches down the road when working on a large application.